setup.rst

```diff
@@ -404,6 +404,10 @@ commands.
 Explaining how to build optional dependencies on a UNIX based system without
 root access is beyond the scope of this guide.
 
+For more details on various options and considerations for building, refer
+to the `macOS readme
+<https://github.com/python/cpython/blob/master/Mac/README>`_.
```

Comment:
Use the new URL with the main branch: https://github.com/python/cpython/blob/main/Mac/README.rst

Reply:
Done